### ISSUE TITLE
defining order of webpages

### ISSUE THREAD

**Issue by Garrafao** [codalab/codalab-competitions#2678]
can the order of webpages in the competition be defined in the competition.yaml?

When I upload our competition and try to define the order of webpages in the admin settings not all pages have an 'order' field and after updating the order several times individual order fields are not displayed anymore so that the ordering cannot be changed. I am using the latest version of Firefox. Thanks for your help in advance!

**Comment by Didayolo** [comment]
Hi,

I am not sure but I think some specific pages are pre-defined order, e.g. "Overview", "Evaluation" or "Rules" pages.

It seems that you have the "order" field only for your **custom webpages**...

This needs to be confirmed.

**Comment by Tthomas63** [comment]
Yes this is the case. The ordering of the default pages is pre-defined. Only custom webpages can have the ordering specified.
